data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg

```diff
@@ -22,21 +22,21 @@
         [/part]
         [part]
-            story= _ "The war council’s thoughts quickly turned to the dwarves, close allies in the past. However, nobody knew the state of things in the Knalgan tunnels. The bravest of scouts found in there the spoor of many trolls, and of creatures darker than trolls. It was even rumored that the dwarven dead from the invasion of Knalga had risen in unlife, roaming the tunnels to slay the living. Fighting through all manner of unknown monsters in the mere hope of finding surviving dwarves held but little appeal."
+            story= _ "The war council’s thoughts quickly turned to the dwarves, close allies in the past. However, nobody knew the state of things in the Knalgan tunnels. The bravest of scouts found in there the spoor of many trolls, and of even viler creatures. For years rumors had spread of the dwarves, slain in the invasion of Knalga, now risen in unlife to roam the tunnels. To Tallin and the others, fighting through all manner of unknown monsters in the mere hope of finding surviving dwarves held but little appeal."
         [/part]
         [part]
             # wmllint: local spelling woodsrunners
-            story= _ "So the matter lay, and the peasants dithered from one suggested plan of action to the next, never arriving at any conclusion. In the meantime, some help, unlooked for, arrived in the form of a small band of woodsrunners — escapees from the orcs, grown wood-wise and crafty, and in contact with larger bands of poachers and outlaws roaming the night. Embassies were sent; agreements made. A handful of experienced fighters, scarred and closemouthed, came to Dwarven Doors to find and train the most willing in their rugged fighting style."
+            story= _ "So the matter lay. The peasants dithered from one suggested plan of action to the next, never arriving at any conclusion. However, some unlooked help arrived in the form of a small band of woodsrunners — escapees from the orcs, grown wood-wise and crafty, and in contact with larger bands of poachers and outlaws roaming the night. Embassies were sent; agreements made. A handful of experienced fighters, scarred and closemouthed, came to Dwarven Doors to find and train the willing in their rugged fighting style."
         [/part]
         [part]
-            # wmllint: local spelling nightrunners
-            story= _ "A fortnight after their victory, the town was awakened from slumber in the dead of night by the distant booming of orcish drums. Scouts training with the nightrunners returned to cry the news that a strong orcish company was approaching. Frightened as they had never been before, the folk of Dwarven Doors looked to Tallin for guidance."
+            # wmllint: local spelling woodsrunners
```

Comment:
A "local spelling" continues until the end of the file, so this isn't needed. It generates a warning about the duplicate declaration.